PhysicsTools/PatAlgos/python/patInputFiles_cff.py

```diff
@@ -24,7 +24,7 @@
                         #, numberOfFiles = 1
                         #, useDAS        = True
                         #)
-        '/store/relval/CMSSW_8_0_0/RelValProdTTbar_13/AODSIM/80X_mcRun2_asymptotic_v4-v1/10000/DE81ABBF-1DDA-E511-8AF8-0026189438B5.root'
+        '/store/relval/CMSSW_9_1_0_pre2/RelValTTbar_13/GEN-SIM-RECO/90X_upgrade2017_realistic_v20-v2/00000/2257937F-3019-E711-BF48-0CC47A4D7678.root'
```

Comment:
it seems more appropriate to define a new vstring for phase1 or otherwise also uniformly migrate PhysicsTools/PatAlgos/python/patTemplate_cfg.py to use 2017 GT and era